Adding update for redis lists

tbickford opened the pull request:
This pertains to the PR I have made for adding the optional authentication params for the Redis Scaler (https://github.com/kedacore/keda/pull/962).  Added some additional wording around the use of those attributes as env vars. I added this to the `2.0` version and created a new version `1.6` as I would think it would go out with the next release? Is there another branch I should be using as the base besides the master branch for potential adds in the next release?

tomkerkhove commented on content/docs/2.0/scalers/redis-lists.md:
```diff
@@ -39,6 +39,14 @@ The `databaseIndex` parameter let the user select the redis database to use. If
+**Connection Authentication:**
+
+- `address` - The host and port for the Redis server (host:port format).
+
```
```suggestion
```

tomkerkhove commented on content/docs/2.0/scalers/redis-lists.md:
```diff
@@ -39,6 +39,14 @@ The `databaseIndex` parameter let the user select the redis database to use. If
+- `address` - The host and port for the Redis server (host:port format).
+
+- `host` - The host of the Redis server. If specified, the `port` should also be specified
+
```
```suggestion
```

tomkerkhove commented on content/docs/2.0/scalers/redis-lists.md:
```diff
@@ -39,6 +39,14 @@ The `databaseIndex` parameter let the user select the redis database to use. If
+
+- `address` - The host and port for the Redis server (host:port format).
+
+- `host` - The host of the Redis server. If specified, the `port` should also be specified
```
```suggestion
- `host` - The hostname of the Redis server. If specified, the `port` should also be specified
```

tomkerkhove commented on content/docs/2.0/scalers/redis-lists.md:
```diff
@@ -39,6 +39,14 @@ The `databaseIndex` parameter let the user select the redis database to use. If
 
+**Connection Authentication:**
+
+- `address` - The host and port for the Redis server (host:port format).
```
```suggestion
- `address` - The hostname and port for the Redis server (host:port format).
```

zroubalik commented:
@tomkerkhove @tbickford what is the status of this?

tbickford commented:
> @tomkerkhove @tbickford what is the status of this?

I thought this was good to go or did I miss something?

tomkerkhove commented:
I've added some suggestions above but they are nits, for the rest I'm fine.

Up to you folks.

tbickford commented:
> I've added some suggestions above but they are nits, for the rest I'm fine.
> 
> Up to you folks.

My bad - I had thought I addressed those items. I pushed an update.
